### ISSUE TITLE
New area for userland migrations

### ISSUE THREAD

**Issue by JakobJingleheimer** [nodejs/nodejs.org#7267]
### Enter your suggestions in details:

The new userland migrations needs a place on nodejs.org to publicise its existence and explain how to use it. Each API doc may reference a specific migration recipe (probably a link to github), but the overall concept should have its own place. Perhaps a Learn section for authorship and consumption?

**Comment by ovflowd** [comment]
Can you elaborate more?

**Comment by AugustinMauroy** [comment]
I'm not 100% sure that good idea to put it in learn but idk where we can put that. Maybe a new subsection on about but not really right

**Comment by JakobJingleheimer** [comment]
> Can you elaborate more?

In what regard? Ex do you want to know what "userland migrations" are, or are you wondering about the content for the new website section?

**Comment by bmuenzenmeyer** [comment]
it'd be kinda nice to include within:

- changelogs / releases
- within https://nodejs.org/en/about/previous-releases#looking-for-latest-release-of-a-version-branch

perhaps you've given it more thought too @JakobJingleheimer but i am a big +1 to provide this in as many places people will see it

here's some MUI examples

inline within relevant docs

![image](https://github.com/user-attachments/assets/effc663b-f487-4328-a2be-daf8baff37db)

upgrade posts

https://mui.com/material-ui/migration/upgrade-to-v6/#breaking-changes

**Comment by JakobJingleheimer** [comment]
Yes! If the codemod is available when the feature is released (the ideal case), I think it should definitely be included in the release notes / changelog 🙂

**Comment by ovflowd** [comment]
I believe Brian replied to the questions I had (unintentionally); I wasn't sure what this was about, but having little info boxes across pages mentioning these makes sense to me :)

**Comment by JakobJingleheimer** [comment]
> I believe Brian replied to the questions I had (unintentionally); I wasn't sure what this was about, but having little info boxes across pages mentioning these makes sense to me :)

Including in the changelog is not the request (that has nothing to do with the website team 😜).

Userland Migrations is what I presented in Dublin. See the readme for an explanation of what it is: https://github.com/nodejs/userland-migrations/tree/chore/initial-setup

**Comment by mikeesto** [comment]
If I'm understanding correctly - what @JakobJingleheimer is proposing is a new page (or pages) on Learn that explain what codemods are and how to use them. That sounds OK to me.

**Comment by AugustinMauroy** [comment]
Yeah @mikeesto I think you got right. But my question is, is the right place ?

**Comment by JakobJingleheimer** [comment]
Yes, exactly :) Learn was just a starting-point / suggestion.
